data/policy_permissions_list/master_policy_permissions_list.json

```
@@ -2190,7 +2190,11 @@
     {
       "id": "./cost/google/cloud_sql_idle_instance_recommendations/google_sql_idle_instance_recommendations.pt",
       "name": "Google Idle Cloud SQL Instance Recommender",
+<<<<<<< HEAD
+      "version": "4.0",
+=======
       "version": "3.1",
+>>>>>>> master
```

```suggestion
      "version": "4.0",
```
There's a chance this might cause a conflict in the Changelog file too if the version changed to 3.1 elsewhere ☹